mutters-core/src/main/java/com/rabidgremlin/mutters/core/bot/BotResponse.java

```diff
@@ -29,28 +31,24 @@
   /** List of suggested replies that the user could use. */
   private List<String> quickReplies;
 
-  /** Map of debug values. Added by particular bot implementation. */
-  private Map<String, Object> debugValues;
-
   /**
    * Constructor.
    * 
-   * @param response The response text.
-   * @param hint Any hint text for the client to display.
-   * @param askResponse True if this the bot is expecting a further response from the user.
-   * @param attachments The attachments for the response.
+   * @param response     The response text.
+   * @param hint         Any hint text for the client to display.
+   * @param askResponse  True if this the bot is expecting a further response from
+   *                     the user.
+   * @param attachments  The attachments for the response.
    * @param quickReplies List of suggested replies that the user could use.
-   * @param debugValues Map of debug values. Specific to bot implementation.
    */
-  public BotResponse(String response, String hint, boolean askResponse,
-    List<BotResponseAttachment> attachments, List<String> quickReplies, Map<String, Object> debugValues)
+  public BotResponse(String response, String hint, boolean askResponse, List<BotResponseAttachment> attachments,
+      List<String> quickReplies)
   {
-    this.response = response;
+    this.response = Objects.requireNonNull(response);
     this.hint = hint;
```

Comment:
any reason these can be null? especially attachments/quick reply. They could just use empty list

Reply:
Figured it would be too many changes with not that much payoff.